packages/twenty-front/src/modules/settings/security/components/SettingsSSOIdentitiesProvidersListCard.tsx

```diff
@@ -13,7 +13,9 @@ import styled from '@emotion/styled';
 import { useRecoilValue } from 'recoil';
 import { IconKey } from 'twenty-ui';
 
-const StyledLink = styled(Link)<{ isDisabled: boolean }>`
+const StyledLink = styled(Link, {
+  shouldForwardProp: (prop) => prop !== 'isDisabled',
```

Comment:
Could you also add `&& isPropValid(prop)` ? Thanks!

Reply:
Done